### ISSUE TITLE
Redirect to 0.0.0.0 instead of 127.0.0.1 for speed

### ISSUE THREAD

**Issue by jtechpro** [ggsava/block-this#12]
Since localhost takes some time than 0.0.0.0 (which is termed as invalid address), the apps/browsers can render it immediately.

It could increase the performance.

**Comment by ggsava** [comment]
@jtechpro Yes, you are correct. I was already on my todo list. I'm trying to release a completely new version this week and working hard on that, but after 3.0 is released I will address this as well (probably next week). Thanks.
